### PR TITLE
Update the remove statement page

### PR DESCRIPTION
**Pull request by AlexFrid**
Added example usage to the remove page.

### CODE REVIEW

**Review by kearfy — APPROVED** [review]
LGTM

**Comment by Ekwuno** [comment]
Question: I recall chatting about the REMOVE command with @kearfy. In some instances don't you have to update in other to see the changes after removing a record? Should we highlight this here?

File: app/snippets/docs/surrealql/statements/remove/example.surql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+REMOVE SCOPE documentation;
+
+REMOVE TABLE article;
+
+REMOVE EVENT new_post ON TABLE article;
+
+REMOVE FUNCTION fn::update_author;
+
+REMOVE FIELD tags ON TABLE article;
```
Comment (Ekwuno) [inline app/snippets/docs/surrealql/statements/remove/example.surql:17]: Question: I recall chatting about the REMOVE command with @kearfy. In some instances don't you have to update in other to see the changes after removing a record? Should we highlight this here?

Comment (AlexFrid) [reply app/snippets/docs/surrealql/statements/remove/example.surql:17]: Yeah, I think we should explain that in the guides instead of here.

**Pull request merged**